packages/block-editor/src/components/link-control/README.md

```diff
@@ -70,37 +111,41 @@ Whether to present initial suggestions immediately.
 - Type: `boolean`
 - Required: No
 
-If passed as either `true` or `false`, controls the internal editing state of the component to respective show or not show the URL input field.
+If passed as either `true` or `false`, controls the internal editing state of the component to respectively show or not to show the URL input field.
 
 
 ### createSuggestion
 
 - Type: `function`
 - Required: No
+- Returns: when called may return either a new `suggestion` directly or a `Promise` which resolves to a
```

Comment:
Other comments have used a capital letter on the beginning of the comment, ie "Returns: When..." instead of "Returns: when..."